navigation/libandroid-navigation-ui/src/main/res/layout/turn_lane_listitem_layout.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:gravity="center"
+    android:layout_marginTop="4dp"
+    android:layout_marginBottom="4dp"
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you <kbd>command</kbd><kbd>alt</kbd><kbd>l</kbd> all your xml files, the formatting is off

Reply:
Updated to latest v2 guidance style